dev/Vagrantfile

```diff
@@ -0,0 +1,16 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.hostname = "vermont"
+  config.vm.box = "trusty_amd"
+  config.vm.box_url = "https://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-amd64-vagrant-disk1.box"
```

Comment:
Any particular reason you chose a trusty64 box? Would Xenial, which is also LTS perhaps be preferable?
Both of the above boxes only support the virtualbox provider. Would perhaps using https://app.vagrantup.com/debian/boxes/jessie64 which supports several providers allow people that want to use say `vagrant up --provider=libvirt`?

Reply:
Thank you! I used the trusty64 since travis uses this setup. I am happy to change it so something more preferable.

Reply:
Changing to jessie to allow for mor virtualization providers sounds good to me 👍

Reply:
Migrated to debian/contrib-jessie64, since the original debian/jessie64 box does not contain guest additions for virtualbox. The box should support other providers as well. 
@nickbroon: Fine by you?

Reply:
Yes, debian/contrib-jessie64 is fine.

The normal way to work with debian/jesssie54 on virtualbox is not `vagrant plugin install vagrant-vbguest` which will then take care of installing the guest additions for you.